docs/userGuide/deployingTheSite.md

```diff
@@ -329,10 +329,172 @@ Here are the steps to set up Netlify:
 
 Now your site will be deployed on Netlify at the given address specified after deployment. It will be updated automatically when the default branch of your repo is updated.
 
-Additionally, **when contributors make a pull request to your GitHub repo, you can _preview_ the updated site** at the bottom of the pull request by clicking on `details` link in the PR:
+## Previewing Pull Requests for MarkBind sites
+
+There may be more than one user making changes to your MarkBind site. In such cases, it might be prudent to have a workflow that is centered around pull requests (PRs) such as a [feature branch workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) or a [forking workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). 
```

Comment:
Let's preface a little bit more who this section is for (users having MarkBind projects on github)

Reply:
Okay I have added an info box below `## Previewing Pull Requests for MarkBind sites`